CMake/Assets.cmake

```diff
@@ -177,6 +177,24 @@ if(NOT USE_SDL1 AND NOT VITA)
     ui_art/menu.png)
 endif()
 
+if(BUILD_TESTING)
+  list(APPEND devilutionx_assets

```

Comment:
It'd be better to place these in a separate directory, e.g. `test_assets`, so that they don't accidentally end up in a release.

They can then be opened using `SDL_RWFromFile` directory rather than going via `OpenAsset`.

Reply:
Also, in the repo, they should probably live in `test/assets` or similar (instead of `Packaging/resources/assets`) and they should probably be set up in `test/CMakeLists.txt` (instead of `Assets.cmake`).

Reply:
I moved them to `test/fixtures` and `test/CMakeLists.txt`. 🙂